src/entity/user/UserPersonalizedDigest.ts

```diff
@@ -7,6 +7,11 @@ export enum UserPersonalizedDigestSendType {
   workdays = 'workdays',
 }
 
+export enum UserPersonalizedDigestType {
+  digest = 'digest',
+  reading_reminder = 'reading_reminder',
```

Comment:
We always use CamelCase for enum naming (I see the above is actually wrong)
 
```suggestion
  Digest = 'digest',
  ReadingReminder = 'reading_reminder',
```

Reply:
Waiting for this: https://github.com/dailydotdev/daily-api/pull/1785/files#r1545581484
before I can apply the change. Otherwise it would mess the GQL enum. So i prefer to keep it as it is for now and then fix it. Another alternative is to copy this implementation to my PR as well and fix it here already. Wdyt?

Reply:
Merged